OrbitGl/App.cpp

```diff
@@ -522,6 +537,8 @@ void OrbitApp::MainTick() {
   Capture::Update();
   GTcpServer->MainThreadTick();
 
+  GOrbitApp->main_thread_executor_->ConsumeActions();
```

Comment:
I'd move this next to the ProcessMainThreadCallbacks() calls above as is it conceptually the same thing unless there is a specific reason why it's called here!?

Reply:
done